### PR TITLE
Move Docker install to Docker repo

### PR DESCRIPTION
**Pull request by 3alpha**
- Deleting old way of installing docker and adding docker repo to the install
- Dropping support for old Debian versions

### CODE REVIEW

**Comment by 3alpha** [comment]
- Tested on clean install of Raspberry PI OS ✅

**Review by pablomendezroyo — APPROVED** [review]

File: scripts/dappnode_install_pre.sh [hunk 1]
```diff
@@ -4,19 +4,8 @@
...
-DOCKER_PATH="${DAPPNODE_DIR}/bin/docker/${DOCKER_PKG}"
-DOCKER_CLI_PATH="${DAPPNODE_DIR}/bin/docker/${DOCKER_CLI_PKG}"
-CONTAINERD_PATH="${DAPPNODE_DIR}/bin/docker/${CONTAINERD_PKG}"
-DCMP_PATH="/usr/local/bin/docker-compose"
-DOCKER_URL="${DOCKER_REPO}/${DOCKER_PKG}"
-DOCKER_CLI_URL="${DOCKER_REPO}/${DOCKER_CLI_PKG}"
-CONTAINERD_URL="${DOCKER_REPO}/${CONTAINERD_PKG}"
-DCMP_URL="https://github.com/docker/compose/releases/download/v2.5.0/docker-compose-linux-x86_64"
-WGET="wget -q --show-progress --progress=bar:force"
+lsb_dist="$(. /etc/os-release && echo "$ID")"
```
Comment (pablomendezroyo) [inline scripts/dappnode_install_pre.sh:7]: What if this commands returns a non-expected value, like someone is trying to install dappnode on a non-debian based machine. There should be kind of a check for the value output. But maybe its for a different issue

Comment (3alpha) [reply scripts/dappnode_install_pre.sh:7]: You're right, we should definitely check for that somewhere! Probably right place for that is at the very beginning.

**Comment by dsimog01** [comment]
I have tested the branch and it worked fine for me

**Pull request merged**